website/docs/docs/advanced-usage/components.md

```diff
@@ -153,6 +153,39 @@ The `<SandpackPreview />` component also allows you to add additional buttons to
 ### Additional content
 For advanced use cases, children of `<SandpackPreview>` are rendered at the end of the preview container.
 
+### Getting client instanceSandpackPreviewRef
```

Comment:
```suggestion
### Getting the client instance with SandpackPreviewRef
```

Not sure if that makes sense, but the title did not compute with me considering that the code snippet uses `SandpackPreviewRef`

Reply:
```suggestion
### Getting the client instance from SandpackPreview
```

Reply:
Reads good!